lambda/health_package/metric-settings.json

```diff
@@ -46,38 +248,410 @@
+    "Description": "Sum of successful Amazon S3 put commands for backup over sum of all Amazon S3 backup put commands. Kinesis Data Firehose emits this metric when backup to Amazon S3 is enabled"
   },
   {
-    "Namespace": "AWS/Kinesis",
-    "MetricName": "PutRecord.Success",
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DataReadFromKinesisStream.Bytes",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 18000,
+    "Maximum": 180000,
+    "Description": "When the data source is a Kinesis data stream, this metric indicates the number of bytes read from that data stream. This number includes rereads due to failovers."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DataReadFromKinesisStream.Records",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 20,
+    "Maximum": 500,
+    "Description": "When the data source is a Kinesis data stream, this metric indicates the number of records read from that data stream. This number includes rereads due to failovers."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DeliveryToSplunk.Success",
     "Statistic": "Minimum",
     "ComparisonOperator": "LessThanThreshold",
     "EvaluationPeriods": 2,
     "Period": 300,
     "Multiplier": 1,
     "Minimum": 0.99,
-    "Description": "Alert on 1% failure"
+    "Description": "Alert if count goes below 1"
   },
   {
-    "Namespace": "AWS/Kinesis",
-    "MetricName": "GetRecords.IteratorAgeMilliseconds",
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DeliveryToSplunk.Bytes",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 600000,
+    "Description": "The number of bytes delivered to Splunk over the specified time period"
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DeliveryToSplunk.DataAckLatency",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 50,
+    "Maximum": 95,
+    "Description": "The approximate duration (seconds) it takes to receive an acknowledgement from Splunk after Kinesis Data Firehose sends it data. The increasing or decreasing trend for this metric is more useful than the absolute approximate value. Increasing trends can indicate slower indexing and acknowledgement rates from Splunk indexers."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DeliveryToSplunk.DataFreshness",
     "Statistic": "Maximum",
     "ComparisonOperator": "GreaterThanOrEqualToThreshold",
     "EvaluationPeriods": 2,
     "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 68,
```

Comment:
This says we would never set a calculated alarm threshold lower than 68s. That seems odd. So if the calculated alarm threshold based on behaviour were 60s we'd set the alarm threshold to 68s instead.

Reply:
Not sure why I've set that will correct it thanks.